docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+
+### Duplicates Metrics
+
+Data file containing duplicates metrics for aligned CRAM files. Multiple reads that match at the same position in the genome are located and tagged duplicate reads in the CRAM file, where duplicate reads are defined as originating from a single fragment of DNA.
+
+#### File Types
+
+| Filename Pattern          | Description                                | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------------- | ------------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.duplicates_metrics.tgz | Generated by biobambam2 bammarkduplicates2 | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### OxoG Metrics
+
+Data file containing OxoG metrics, metrics quantifying the error rate resulting from oxidative artifacts, for aligned CRAM files.
+
+#### File Types
+
+| Filename Pattern    | Description                          | Analysis Type | Data Category           | Generating Workflow(s) |
+| ------------------- | ------------------------------------ | ------------- | ----------------------- | ---------------------- |
+| \*.oxog_metrics.tgz | Generated by GATK CollectOxoGMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+### Ploidy and Purity Estimation
+
+Data file containing purity and ploidy estimate.
+
+#### File Types
+
+| Filename Pattern     | Description                          | Analysis Type | Data Category           | Generating Workflow(s)     |
+| -------------------- | ------------------------------------ | ------------- | ----------------------- | -------------------------- |
+| \*.ascat_metrics.tgz | Estimated by Sanger ASCAT CNV caller | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Cross Sample Contamination
+
+Data file containing cross sample contamination estimate.
+
+#### File Types
+
+| Filename Pattern             | Description                                                                                                                                     | Analysis Type | Data Category           | Generating Workflow(s)     |
+| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------------------- | -------------------------- |
+| \*.contamination_metrics.tgz | Estimated by Sanger `verifyBamHomChk.pl` script, which provides information to determine whether the sample is possibly contaminated or swapped | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling |
+
+### Genotyped Gender Metrics
+
+Data file containing genotypes comparison results of CRAM files from the same donor, including the fraction of matched genotypes and inferred the donor gender.
```

Comment:
not sure if this is worded correctly?should it be

`Data file containing genotypes comparison results of CRAM files from the same donor, including the fraction of matched genotypes that infers the gender of the donor.`

Reply:
not quite but  i see the issue and fixed it